packages/cydran/src/log/strategy/AbstractLevelStrategy.ts

```diff
@@ -0,0 +1,150 @@
+import { JSType } from "CydranConstants";
+import Level from "log/Level";
+import { Appender } from "log/appender/Appender";
+import { LevelStrategy } from "log/strategy/LevelStrategy";
+import { compositeArray, isDefined } from "util/Utils";
+
+const withStack = (...moreArgs: any): Error | undefined => {
+	const wkps: any[] = moreArgs.length > 0 ? [... moreArgs] : [];
+	const forStackOut: Error | null = (wkps.length > 0 &&  wkps[wkps.length] instanceof Error) ? wkps[wkps.length] : null;
+	return forStackOut;
+}
+
+const resolvemsgBase = (payloadFn: () => string, logLabel: string, pender: Appender): string => {
```

Comment:
1. resolvemsgBase -> resolveMsgBase to be camelcase consistent
2. Typo on "pender"?

Reply:
done